packages/desktop/components/popups/TokenInformationPopup.svelte

```diff
@@ -89,10 +88,7 @@
         </div>
 
         <div class="space-y-3 flex flex-col items-center justify-center">
-            <AssetIcon {asset} chainId={asset.chainId} large />
-            <Text type={TextType.h2} fontWeight={FontWeight.bold}>
-                {getUnitFromTokenMetadata(asset.metadata)}
-            </Text>
+            <AssetTile classes="mb-2 pointer-events-none" onClick={() => {}} {asset} />
         </div>
```

Comment:
I think you can get rid of all this wrapper `div` as now is not doing anything, and just leave it as

```
<AssetTile classes="pointer-events-none" onClick={() => {}} {asset} />
```
without `<div class="space-y-3 flex flex-col items-center justify-center">`

Reply:
Passing an empty `onClick` along with `pointer-events-none` seems like we could update the asset tile and make it also non clickable but I see this will need a bigger change, so we can leave the empty click I think. Just wanted to add the comment so we always try to make good practices with the components and try to make the code future proof 🌷